pilot/pkg/model/telemetry_test.go

```diff
@@ -639,7 +660,7 @@ func TestTelemetryFilters(t *testing.T) {
 			networking.ListenerProtocolHTTP,
 			nil,
 			map[string]string{
-				"istio.stats": "{}",
+				"istio.stats": `{"tcp_reporting_duration":"5s"}`,
```

Comment:
Is the default in istio/proxy 5s? If so, it may be good to not explicitly set it (bloats the xDS)

Reply:
good gatch, we should change the default value in proxy, will improve it.